ed25519/ed25519_verify/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "ed25519_verify"
+version = "0.1.0"
+authors = ["Jared Wasinger <j-wasinger@hotmail.com>"]
+
+[dependencies]
+sha2 = "0.7.1"
```

Comment:
Is this actually using `sha2`?

Reply:
No.  removing it.

Reply:
Nvm yes it does.  It needs the Sha512 struct.